### PR TITLE
SMHE-672 Add dead battery events

### PR DESCRIPTION
**Pull request by kroesctrl**
Signed-off-by: Gerben Kroes <gerben@kroesctrl.nl>

### CODE REVIEW

**Comment by kroesctrl** [comment]
https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-nightly-build/559/

File: osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/MBusEventLog.java [hunk 1]
```diff
@@ -129,10 +129,10 @@ private void initBufferData() {
     this.bufferData.add(Arrays.asList(this.getNextDateTime(), (short) 255));
 
     // Add all events.
-    this.addEvents((short) 100, (short) 106);
```
Comment (abolder) [inline osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/MBusEventLog.java:132]: Moeten deze wel verwijderd worden, want de permanent errors bestaan nog met 106,116,126 en 136?

**Review by abolder — COMMENTED** [review]
Added a single comment for the addEvents

File: osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/MBusEventLog.java (hunk 1 above)
Comment (kroesctrl) [reply osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/MBusEventLog.java:132]: They are not removed. The range is expanded with 1

**Comment by kroesctrl** [comment]
https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-nightly-build/560/

File: osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/MBusEventLog.java (hunk 1 above)
Comment (abolder) [reply osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/MBusEventLog.java:132]: So thats how it works. Then the changes are approved.

**Comment by kroesctrl** [comment]
https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-nightly-build/563/

**Review by abolder — APPROVED** [review]

**Pull request merged**